### ISSUE TITLE
The color contrast ratio of "Supported file format" text is failing and the cc ratio is 2.441:1 which is less than 4.5:1:A11y_ML.NET Model Builder_Scenario_Contrast.

### ISSUE THREAD

**Issue by AnKushSingh05** [dotnet/machinelearning-modelbuilder#3037]
### GitHubTags
#A11ySev2; #A11yTCS;#Model Builder - Machine Learning for .NET Developers;#DesktopApp; #Win32;#BM-MlnetModelBuilder-Win32-March25; #Win32;#WCAG1.4.3;#Contrast (Minimum);#A11yMAS; 

### Environment Details:
Application Name: VS Setup
Edition Windows 11 Enterprise
Version 24H2
OS build 26100.3476
Microsoft Visual Studio Enterprise 2022 (64-bit) - Current
Version 17.13.3

### Repro Steps:
**Prerequisites:**
Install latest Visual Studio 2022 with at least one of the .NET workloads (ASP.NET and web development, .NET desktop development, or .NET Core cross-platform development) and/or the ML.NET Model Builder component selected.

1. Launch visual studio.
2. Create a new .NET project or open an existing .NET project.
3. Right-click on the project in the solution explorer and Add > Machine Learning.
4. Tab till scenario and select a scenario.
5. Select training environment (Select Azure).
6. Tab till next step and press enter, data screen will open.
7. Verify that the color contrast ratio of "Supported file format" text is failing and the cc ratio is 2.441:1 which is less than 4.5:1

### Actual Result:
The color contrast ratio of "Supported file format" text is failing and the cc ratio is 2.441:1 which is less than 4.5:1

### Expected Result:
The color contrast ratio of "Supported file format" text should be greater than or equal to 4.5:1.

### User Impact:
Low vision users will face difficulties if the text color contrast ratio is inadequate, making it hard for them to read the text.

### Attachment:

![Image](https://github.com/user-attachments/assets/30ac3ced-06b8-4a65-bfcd-2b548f6344b3)

**Comment by zewditu** [comment]
@AnKushSingh05  I am not able to repro this 

![Image](https://github.com/user-attachments/assets/0859fd35-1ba6-41cd-b9f3-1578121a569b)

![Image](https://github.com/user-attachments/assets/f66a6bea-f888-4aa1-8b17-938a5d8e41d3)

**Comment by AnKushSingh05** [comment]
Issue is still not fixed in below version
Microsoft Visual Studio Enterprise 18 (64-bit) - main
Version 18.0.0 Preview 1.0 [10613.167.main]

![Image](https://github.com/user-attachments/assets/93efad59-5a46-4250-9d44-0c8203b30d33)

**Comment by AnKushSingh05** [comment]
Issue is still not fixed
Microsoft Visual Studio Enterprise
Channel: Canary
Version: 18.3.0 Insiders [11308.173.main]

<img width="1280" height="720" alt="Image" src="https://github.com/user-attachments/assets/452add66-e738-4936-9b28-7ce296684cf6" />

**Comment by AnKushSingh05** [comment]
Issue is fixed

<img width="1280" height="720" alt="Image" src="https://github.com/user-attachments/assets/7f950b60-c221-447f-8fa1-9a85bbdab5a9" />

Microsoft Visual Studio Enterprise
Channel: Canary
Version: 18.3.0 Insiders [11311.1.main]
